tasking/pulp/tasking/tasks.py

```diff
@@ -309,10 +315,14 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         storage.delete_working_directory()
 
     def _get_parent_arg(self):
-        """Return a dictionary with the parent set if running inside of a Task"""
+        """Return a dictionary with the 'parent' set if running inside of a PulpUserFacingTask"""
```

Comment:
Oh yes it should, this was the old name a few weeks ago. Good catch. updated with the next push.